roles/sysctl/defaults/main.yml

```diff
@@ -31,5 +31,9 @@ sysctl_defaults:
   compute:
     - name: net.netfilter.nf_conntrack_max
       value: 1048576
+  # https://gitlab.com/yaook/operator/-/commit/1e42eda64fac8045baeae1d88e865c5db70cc25b
```

Comment:
This commit only mentions that yaook is using inotify, have you seen issues in our deployment that look similar? might be better to reference an osism issue if there is one

Reply:
I guess this reference is better than nothing anyway, no need to hold up this patch